test/kitchen/test/integration/win-sysprobe-test/rspec_datadog/win-sysprobe-test_spec.rb

```diff
@@ -26,7 +26,7 @@ def check_output(output, wait_thr)
 
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Can you please add WKIT as owners of this file? It is currently owned by Agent Platform, not sure why 
<img width="1167" alt="image" src="https://github.com/DataDog/datadog-agent/assets/45568537/6cd7bd68-016d-418f-8e3e-dfffff972340">